optimum/onnxruntime/optimization.py

```diff
@@ -117,7 +119,7 @@ def export(
 
         # Export the model if it has not already been exported to ONNX IR
         if not onnx_model_path.exists():
-            export(self.tokenizer, self.model, self._onnx_config, self.opset, onnx_model_path)
+            export(self.preprocessor, self.model, self._onnx_config, self.opset, onnx_model_path)
```

Comment:
I think that it would be better to add the keyword arguments on, although the term `preprocessor` doesn't exist for older version of transformers. WDYT?

Reply:
The constraint would be transformers=4.7+ ?

Reply:
Keeping it as it is for the moment to not break backwards compatibility.